pom.xml

```diff
@@ -987,9 +988,10 @@
                         <exclude>${test.files.integration}</exclude>
                     </excludes>
                     <runOrder>random</runOrder>
+                    <parallel>true</parallel>
                     <forkCount>1C</forkCount>
-                    <threadCount>1</threadCount>
                     <reuseForks>true</reuseForks>
+                    <useUnlimitedThreads>true</useUnlimitedThreads>
```

Comment:
If I remeber correctly we used thread count as 1 as Tetss became unpredictable. Not sure though. Can we run tests multiple times  to see correct beaviour?

Reply:
Using only a single thread makes the unit test build very slow, particularly noticeable in whois-internal for some reason. I ran the build multiple times locally and in GitLab and all seems fine.